dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -350,6 +351,68 @@ public Workflow initializeWorkflow(String repositoryId, Workflow workflow) {
         return workflow;
     }
 
+    @Override
+    public Workflow setupWorkflowVersions(String repositoryId, Workflow workflow, Optional<Workflow> existingWorkflow,
```

Comment:
Is there a reason to move this method? It's hard to tell what you changed, if anything, from the way it was.

Reply:
I was getting a puzzling checkstyle test, but forgot to move it back once I figured it out. I didn't change anything.